config.dev.toml

```diff
@@ -99,6 +99,10 @@ EnablePprof = false
 # PprofPort specifies the port on which pprof will be exposed if activated.
 PprofPort = ":3001"
 
+# ConfigPort specifies the port on which the /config endpoint will listen for
+# providing the internal configuration
+#ConfigPort = ":3002"
```

Comment:
Per my comment about making `*string` into a `string`. You can uncomment this line and just do `ConfigPort = ""` similar to a lot of our configuration fields